framework/src/vectorpostprocessors/CSVReader.C

```diff
@@ -41,6 +41,7 @@ CSVReader::validParams()
                                "omitted it will read comma or space separated files.");
   params.addParam<bool>(
       "ignore_empty_lines", true, "When true new empty lines in the file are ignored.");
+  params.suppressParameter<bool>("contains_complete_history");
```

Comment:
Do we know why this was removed? It seems like something we may want for the CSVReader.

Reply:
It seemed like the correct change when I looked at it, but I am not sure why I thought that. It actually should be set to true and suppressed. That would make the output of this match the input. Do you agree?